src/partr.c

```diff
@@ -253,6 +253,8 @@ static jl_task_t *get_next_task(jl_value_t *getsticky)
 JL_DLLEXPORT jl_task_t *jl_task_get_next(jl_value_t *getsticky)
 {
     jl_ptls_t ptls = jl_get_ptls_states();
+    // avoid blocking for I/O immediately, in case @threads is about to exit
```

Comment:
don't do this—it'll have little effect on the correctness of code, but just make the related issues harder to fix

Reply:
Well, ok, but I figured we'd have a threshold like this eventually anyway.

Reply:
oh, do we not have any delay right now? From the comment, I thought the reason you were trying to avoid blocking was because of `@threads`. It sounds instead that what you meant was to avoid blocking too quickly after draining the workqueue to watch for new items.

Reply:
Yes, that would be a better way to describe it.